libs/cardano-ledger-binary/src/Cardano/Ledger/Binary/Encoding/EncCBOR.hs

```diff
@@ -1147,6 +1147,8 @@ instance EncCBOR BlockNo where
   encCBOR = fromPlainEncoding . Serialise.encode
   encodedSizeExpr size = encodedSizeExpr size . fmap unBlockNo
 
+deriving instance EncCBOR EpochInterval
```

Comment:
As you already found out from changes to CHaPs, this change will require a lower bound update on `cardano-slotting`